rasa_core/policies/keras_policy.py

```diff
@@ -56,13 +62,14 @@ def __init__(self,
         self.current_epoch = current_epoch
 
     def _load_params(self, **kwargs: Dict[Text, Any]) -> None:
-        config = copy.deepcopy(self.defaults)
-        config.update(kwargs)
+        self.config = copy.deepcopy(dict(self.defaults, **self.tf_defaults))
```

Comment:
since we persist only `tf_config` now in both policies, no need for `self.` here. We needed it in previous iterations where the whole config was persisted
```suggestion
        config = copy.deepcopy(dict(self.defaults, **self.tf_defaults))
```